reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -150,8 +150,20 @@ data class EvaluatedModel(
         fun create(input: ReporterInput): EvaluatedModel = EvaluatedModelMapper(input).build()
     }
 
-    fun toJson(writer: Writer): Unit = JSON_MAPPER.writerWithDefaultPrettyPrinter().writeValue(writer, toSortedTree())
+    /**
+     * Serialize this [EvaluatedModel] to JSON and write the output to the [writer], optionally
+     * [pretty printed][prettyPrint].
+     */
+    fun toJson(writer: Writer, prettyPrint: Boolean) {
```

Comment:
Alternatively:

    fun toJson(writer: Writer, prettyPrint: Boolean): Unit =
        JSON_MAPPER.apply {
            if (prettyPrint) writerWithDefaultPrettyPrinter() else writer()
        }.writeValue(writer, toSortedTree())

Reply:
I don't find this easier to read.